index.js

```diff
@@ -251,6 +252,8 @@ module.exports = (command, args, options) => {
 				resolve({error});
 			});
 		}
+
+		oncancel(parsed.options.oncancel);
```

Comment:
This should kill the process, not execute a function from an option.

Reply:
done